src/engine/strat_engine/pool.rs

```diff
@@ -139,17 +139,26 @@ fn check_metadata(metadata: &PoolSave) -> StratisResult<()> {
 // Takes a set of information determined about the pool in liminal devices and
 // determines what the state of the pool should be when it is set up.
 fn get_pool_state(info: Option<PoolEncryptionInfo>, backstore: &Backstore) -> ActionAvailability {
-    let avail = if let Some(i) = info {
-        if i.is_inconsistent() {
-            warn!("Metadata for encryption inconsistent across devices in pool");
-            ActionAvailability::NoRequests
+    let state = max(
+        if let Some(i) = info {
+            if i.is_inconsistent() {
+                warn!("Metadata for encryption inconsistent across devices in pool");
+                ActionAvailability::NoRequests
+            } else {
+                ActionAvailability::Full
+            }
         } else {
             ActionAvailability::Full
-        }
-    } else {
-        ActionAvailability::Full
-    };
-    max(avail, backstore.action_availability())
+        },
+        backstore.action_availability(),
+    );
+    if state != ActionAvailability::Full {
```

Comment:
There is also a log message emitted in ```StratPool::setup()```.  One of these messages might be considered superfluous; you decide.

Reply:
You're right! Resolved.